api/features/workflows/core/models.py

```diff
@@ -91,6 +91,8 @@ class ChangeRequest(
         null=True,
     )
 
+    ignore_conflicts = models.BooleanField(default=False)
```

Comment:
Do we need to add this to some serializer to make it editable from the UI?

Reply:
Yes, in the workflows repo which I will do after merging this PR so I can test it there properly.